src/code/z_player_lib.c

```diff
@@ -654,7 +654,22 @@ u8 sEyeMouthIndexes[][2] = {
  * This allows the array to only contain the symbols for one file and have it apply to both. This is a problem for
  * shiftability, and changes will need to be made in the code to account for this in a modding scenario. The symbols
  * from adult Link's object are used here.
+ *
+ * Using AVOID_UB here will use a 2D array instead and properly use the child link pointers to allow for shifting.
  */
+#ifdef AVOID_UB
```

Comment:
Could we put avoid_ub things inside the if(n)defs, so hypothetical processing of the source could "get rid" of all avoid_ub related things
```suggestion
 */
#ifdef AVOID_UB
// Using AVOID_UB here will use a 2D array instead and properly use the child link pointers to allow for shifting.
```